collection-scripts/gather_crs

```diff
@@ -8,33 +8,33 @@ if [[ -z "$DIR_NAME" ]]; then
 fi
 
 # Resource list
-resources=()
+crs=()
 
 # explictly adding rabbit in the grep because it's not part of the openstack
 # umbrella and we rely on the upstream rabbitmq-cluster-operator
 for i in $(/usr/bin/oc get crd | grep -E "(openstack|rabbitmq)\.(org|com)" | awk '{print $1}')
 do
-  resources+=("$i")
+  crs+=("$i")
 done
 
 # explicitly adding Metal3 BareMetalHosts
-resources+=("baremetalhosts.metal3.io")
+crs+=("baremetalhosts.metal3.io")
 
 # we use nested loops to nicely output objects partitioned per namespace, kind
 echo "Gathering CRs"
-for resource in "${resources[@]}"; do
-  /usr/bin/oc get "${resource}" --all-namespaces -o custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace --no-headers 2> /dev/null | \
+for res in "${crs[@]}"; do
+  /usr/bin/oc get "${res}" --all-namespaces -o custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace --no-headers 2> /dev/null | \
   while read -r ocresource; do
     ocobject=$(echo "$ocresource" | awk '{print $1}')
     ocproject=$(echo "$ocresource" | awk '{print $2}')
-    if [ -z "${ocproject}" ]||[ "${ocproject}" == "<none>" ]; then
-      object_collection_path=${BASE_COLLECTION_PATH}/cluster-scoped-resources/${resource}
+    if [ -n "${ocproject}" ]||[ "${ocproject}" != "<none>" ]; then
```

Comment:
-1: This should not be an or (`||`) but an and (`&&`)

Reply:
right, not sure why I have this leftover here, but yes, when I change -z to -n I intended to change the `and` condition as well! Updating it!